Makefile.osx

```diff
@@ -88,11 +89,9 @@ $(ICNS): res/icon16.png res/icon32.png res/icon256.png res/icon512.png
 	cp res/icon512.png  $(ICONSET)/icon_256x256@2x.png
 	cp res/icon512.png  $(ICONSET)/icon_512x512.png
 	cp res/icon1024.png $(ICONSET)/icon_512x512@2x.png
-	
-ifneq ($(ICONUTIL),)
 	$(ICONUTIL) -c icns -o $@ $(ICONSET)
 else ifneq ($(PNG2ICNS),)
-	$(PNG2ICNS) $@ $(filter-out %@2x.png,$(wildcard $(ICONSET)/*.png))
+	$(PNG2ICNS) $@ res/icon{16,32,128,256,512}.png
```

Comment:
Here we could reuse the list of prerequisites instead:

```Makefile
 	$(PNG2ICNS) $@ $^
```

But `res/icon128.png` is missing in that list. I don't know if that is intentional.

Reply:
Unintentional... as is icon1024.png being missing